x-pack/plugins/infra/server/lib/snapshot/snapshot.ts

```diff
@@ -146,6 +146,11 @@ const requestNodeMetrics = async (
               date_histogram: {
                 field: options.sourceConfiguration.fields.timestamp,
                 interval: options.timerange.interval || '1m',
+                offset: '-58s',
```

Comment:
What's going on here? Why `offset` by `-58s`?

Reply:
Good catch -- I forgot to port over the offset calculation from the old `nodes` endpoint. Will fix.